tensorflow/python/ops/math_ops.py

```diff
@@ -3204,6 +3204,16 @@ def sigmoid(x, name=None):
   """Computes sigmoid of `x` element-wise.
 
   Specifically, `y = 1 / (1 + exp(-x))`.
+  
```

Comment:
Just put the usage xample above the args section 😊

Reply:
Oh and remove the blank lines

Reply:
Are you talking about the lines in between sections? I believe they are required for the doctest to properly work (let me know if I'm misinterpreting).

Reply:
Yes I am. I am not sure of what you're saying but I said this because I saw it here https://www.tensorflow.org/community/contribute/docs_ref . You can read it if you're confused(I did) 👍

Reply:
Hi, I have read the document before, and in the example that provided they had an extra line between sections.

Reply:
Yes they did. I had a pull request I made before where I had to remove the spacing. I guess that was a different case. 😊

Reply:
That's fine. I really appreciate you reviewing this request!

Reply:
thanks for understanding! I guess the whole point of gci is to help us figure all this out.